pkg/lib/qunit-tests.ts

```diff
@@ -62,7 +62,7 @@ window.setTimeout(() => {
 /* QUnit-Tap writes the summary line right after this function returns.
 * Delay printing the end marker until after that summary is out.
 */
-QUnit.done(() => window.setTimeout(() => console.log("cockpittest-tap-done"), 0));
+QUnit.done(() => { window.setTimeout(() => console.log("cockpittest-tap-done"), 0) });
```

Comment:
TS doesn't like the return value here? (setTimeout returns a handle for clearing the timer)

Reply:
Yes.  Exactly.